app/models/pension/charges/LifetimeAllowance.scala

```diff
@@ -16,16 +16,34 @@
 
 package models.pension.charges
 
+import models.mongo.TextAndKey
 import play.api.libs.json.{Json, OFormat}
-import utils.EncryptedValue
+import utils.{EncryptedValue, SecureGCMCipher}
+import utils.DecryptableSyntax.DecryptableOps
+import utils.DecryptorInstances.bigDecimalDecryptor
+import utils.EncryptableSyntax.EncryptableOps
+import utils.EncryptorInstances.bigDecimalEncryptor
 
-case class LifetimeAllowance(amount: BigDecimal, taxPaid: BigDecimal)
+case class LifetimeAllowance(amount: BigDecimal, taxPaid: BigDecimal) {
+
+  def encrypted()(implicit secureGCMCipher: SecureGCMCipher, textAndKey: TextAndKey): EncryptedLifetimeAllowance =
```

Comment:
does this need to be implicit ?

Reply:
yes won't work else - how new solution was designed